### PR TITLE
added SOC2 section

### PR DESCRIPTION
**Pull request by natapg**
Closes WEB-209

### CODE REVIEW

File: src/components/Billing/Billing.module.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .section {
   background-image: url('../../img/billing-background.svg');
   background-repeat: no-repeat;
-  background-position: 110% 200%;
+  background-position: 110% 100%;
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/Billing/Billing.module.scss:4]: Kate is fine with this change

File: src/components/SOCSection/SOCSection.module.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    font-size: rem(40px);
+    font-weight: v('bold');
+    line-height: rem(53px);
+
+    padding: 0 rem(6px) rem(20px) rem(6px);
+
+    @include media('<=phoneLandscape') {
+      font-size: rem(27px);
+      line-height: 120%;
+      letter-spacing: -0.01em;
```
Comment (Lisenish) [inline src/components/SOCSection/SOCSection.module.scss:30]: Looks strange, why?

File: src/components/SOCSection/SOCSection.module.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  }
+}
+.cardsContainer {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+
+  --contentWidth: 745px;
+
+  @include media('<=phoneLandscape') {
+    display: flex;
```
Comment (Lisenish) [inline src/components/SOCSection/SOCSection.module.scss:51]: Wouldn't it be easier just to change grid to 1 column layout?

File: src/components/SOCSection/SOCSection.module.scss [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      box-shadow: 0px 8px 32px rgba(13, 16, 43, 0.1);
+      border-radius: rem(16px);
+
+      @include media('<=phoneLandscape') {
+        margin-bottom: rem(16px);
+      }
+    }
+
+    .cardTitle {
+      font-family: v(ff-sans);
```
Comment (Lisenish) [inline src/components/SOCSection/SOCSection.module.scss:79]: We probaby need to create mixins for typograghy

File: src/components/SOCSection/SOCSection.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  )
+}
+
+interface CardProps {
+  icon: React.ReactNode
+  title: string
+}
+function Card({ icon, title }: CardProps) {
+  return (
+    <div className={styles.card}>
```
Comment (Lisenish) [inline src/components/SOCSection/SOCSection.tsx:36]: Maybe these can be more semantic?

**Review by Lisenish — APPROVED** [review]

File: src/components/SOCSection/SOCSection.module.scss (hunk 3 above)
Comment (natapg) [reply src/components/SOCSection/SOCSection.module.scss:51]: Yes, it's easier, you're right.

File: src/components/SOCSection/SOCSection.module.scss (hunk 2 above)
Comment (natapg) [reply src/components/SOCSection/SOCSection.module.scss:30]: Do you mean negative letter spacing?

**Pull request merged**